modules/programs/i3status-rust.nix

```diff
@@ -8,6 +8,16 @@ let
 
   settingsFormat = pkgs.formats.toml { };
 
+  # Since 0.31, the "block" key has to be first in the TOML output.
+  prefixBlockType = toml:
+    toml.overrideAttrs (oldAttrs: {
+      nativeBuildInputs = oldAttrs.nativeBuildInputs ++ [ pkgs.jq ];
+      buildCommand = ''
+        cat "$valuePath" \
+          | jq '.block |= map({block: .block} + del(.block))' \
+          | json2toml -p > "$out"
```

Comment:
The use of `cat` is unnecessary here.

Also, rather than using `overrideAttrs` and coupling the module to the internals of `pkgs.formats.toml.generate`, I think the derivation is short enough that it's better to recreate it from scratch.